src/pranitaPJun20/ArmstrongNumber.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Program 4: WAP to check if a number is Armstrong or not.
+Example:-
+Input:- 153
+Output:-153
+hint:- (1*1*1)+(5*5*5)+(3*3*3)
+1+125+27
+153
+ */
+package pranitaPJun20;
+
+public class ArmstrongNumber {
+	//Method to find Armstrong Number
+	void findArmstrongNumber(int input) {
+		int numExpected= input;
+		int numActual=0;
+		while(input>0) {
+			int lastDigit  = input%10;
+			input=input/10;
+			numActual = numActual+(lastDigit*lastDigit*lastDigit);
```

Comment:
**Suggestion:**
You can take a look at power function - Math.pow(x,y)
which will help you achieve same result for multiplication of same number multiple times (to find square, cube and so on).
i.e. Math.pow((number % 10), 3))

Reply:
Thank you Aditi.I came to know about new function.Changes Done.